packages/core/src/components/block/block.stories.tsx

```diff
@@ -92,8 +111,20 @@ Nested.argTypes = {
       defaultValue: { summary: 'Primary' },
     },
   },
+  componentTag: {
+    name: 'Component Tag',
+    description: 'Specifies the HTML tag used for the component wrapper.',
+    control: {
+      type: 'radio',
+    },
+    options: ['section', 'div', 'article', 'aside', 'header', 'footer', 'nav', 'main'],
+    table: {
+      defaultValue: { summary: 'article' },
+    },
+  },
 };
 
 Nested.args = {
   outerModeVariant: 'Primary',
+  componentTag: 'article',
```

Comment:
Let's make this one reflect the components prop default state: 
```suggestion
  componentTag: 'div',
```